daemon/src/db_sql/query_load_all_cfds.sql

```diff
@@ -0,0 +1,69 @@
+with order as (
```

Comment:
I'd prefer if the changes were made in two steps:

1. Moving queries to external files
2. Changing up the query to use `with`